pkgs/jupiter-hw-support/firmware.nix

```diff
@@ -103,6 +103,12 @@ stdenv.mkDerivation {
     wrapProgram $out/bin/jupiter-controller-update \
       --prefix PATH : ${lib.makeBinPath [ jq pythonEnv ]}
 
+    mkdir -p $out/libexec
+    makeShellWrapper \
+      $out/share/jupiter_controller_fw_updater/d20bootloader.py \
+      $out/libexec/d20bootloader \
+      --prefix PATH : ${lib.makeBinPath [ jq pythonEnv ]}
+
```

Comment:
This does not add the wrapper in `bin` so they don't get accidentally played with.

This also would allow one to use the "fun" features more trivially, considering it needs wrapping to work well enough.